jobs/kube-apiserver/templates/config/audit_policy.yml.erb

```
@@ -0,0 +1 @@
+<% require 'yaml' %><%= p('audit-policy').to_yaml %>
```

Could you make the policy optional, so operator don't have to fill it in, if they don't use audit logging.
```suggestion
<% require 'yaml' %><%= p('audit-policy','').to_yaml %>
```

---

Done.